### PR TITLE
feat: Import multisig identity during account import

### PR DESCRIPTION
**Pull request by patnir**
## Summary

When importing a multisig account to a new datadir, the secret is likely to not exist in this data dir. so that means the encrypted communication components of the multisig commands cannot be run. 

This PR changes account import to look for the identity, if it doesn't exist then to import the identity

## Documentation

Does this change require any updates to the Iron Fish Docs (ex. [the RPC API
Reference](https://ironfish.network/docs/onboarding/rpc/chain))? If yes, link a
related documentation pull request for the website.

```
[ ] Yes
```

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and label it with `breaking-change-rpc` or `breaking-change-sdk`.

```
[ ] Yes
```

### CODE REVIEW

**Review by hughy — APPROVED** [review]
is there an easy way to unit test this?

File: ironfish/src/wallet/wallet.ts [hunk 1]
```diff
@@ -1433,12 +1436,26 @@ export class Wallet {
...
+      secret = Buffer.from(accountValue.multisigKeys.secret, 'hex')
     }
 
     if (name && this.getAccountByName(name)) {
       throw new DuplicateAccountNameError(name)
     }
 
+    if (secret) {
+      const duplicateSecret = await this.walletDb.getMultisigSecretByName(accountValue.name)
+      if (duplicateSecret && duplicateSecret.equals(secret)) {
```
Comment (hughy) [inline ironfish/src/wallet/wallet.ts:1452]: should this be `&& !duplicateSecret.equals(secret)`?

Comment (patnir) [reply ironfish/src/wallet/wallet.ts]: updated the logic here + test

File: ironfish/src/wallet/wallet.ts [hunk 2]
```diff
@@ -1492,6 +1515,24 @@ export class Wallet {
         await this.walletDb.setAccount(account, tx)
       }
 
+      if (secret) {
+        const identitySerialized = new multisig.ParticipantSecret(secret)
+          .toIdentity()
+          .serialize()
+        const multisigIdentity = await this.walletDb.getMultisigIdentity(identitySerialized)
```
Comment (hughy) [inline ironfish/src/wallet/wallet.ts:1522]: We're doing these operations (deserializing the secret, deriving the identity, reading from the db) twice in this method for the same secret. Can we combine these checks into one inside the db transaction?

Comment (patnir) [reply ironfish/src/wallet/wallet.ts]: Done, I thought there might a slight downside to doing it in the transaction, but its actually better to do so. i'm passing the tx to all the getter methods which will provide a better experience incase there is another db transaction touching that secret

**Pull request merged**